### PR TITLE
Removes "fat" status and everything related.

### PR DESCRIPTION
**Pull request by lizardqueenlexi**
## About The Pull Request

As the title says, eating too much no longer makes you "fat". You suffer no slowdown or mood debuff from eating too much, and in general, the game will not take every opportunity to make fun of you.

Notable removals/changes:
- The "fat sucker" machine is totally gone.
- Shady Slim's cigarettes have been removed (since they only existed to interact with this mechanic).
- Lipoplasty surgery is gone.
- The nutrition setting of scanner gates is gone.

There are a couple of other removals too, like Gluttony's Wall, that I think were already not in use on this codebase.

One thing I'm not completely satisfied with was the change to mint toxin, which now does quite literally nothing. I don't think this is especially a problem, it just makes its existence a bit vestigial.

Also includes an UpdatePaths script to delete all removed objects, just in case.

## Why It's Good For The Game

image

Basically, removes some shitty "jokes" about fat people. It's an extremely mean-spirited feature that serves no actual purpose, and punishes you for clicking on a burger one time too many. It could potentially be replaced later with a less mean-spirited "overeating" mechanic, but I'm dubious as to whether that would be any fun either.

## Changelog

:cl:
del: Removed the "fat" status - overeating now has no negative effects.
del: Removed lipoplasty surgery.
del: Removed the fat sucker machine.
tweak: Scanner gates no longer have a "nutrition" option available.
/:cl:

### CODE REVIEW

**Comment by rye-rice** [comment]
🙏 
bless you

**Review by thgvr — APPROVED** [review]

**Comment by Imaginos16** [comment]
Unfathomably based, thank you for your service.

**Comment by CoiledLamb** [comment]
thank you lexi!!!!

**Review by MarkSuckerberg — APPROVED** [review]
Thank you

**Comment by thgvr** [comment]
I'm fine with removing all this even without some mechanic to limit just eating 300 burgers and being fine all round, it's a bit of a relic and annoying anyways.

**Comment by ZephyrTFA** [comment]
bad pr, host veto
